sdk/storage/Azure.Storage.Common/src/Shared/StorageResponseClassifier.cs

```diff
@@ -41,6 +41,19 @@ public override bool IsRetriableResponse(HttpMessage message)
                         return true;
                 }
             }
+
+            // Retry select Copy Source Error Codes
+            if (message.Response.Headers.TryGetValue(Constants.HeaderNames.CopySourceErrorCode, out string copySourceError))
```

Comment:
@amnguye @jaschrep-msft @jalauzon-msft @tg-msft, what is the most straightforward way to test this?  It's non-trivial to get the service to return these copy source error codes, so the test will probably need to involve mocking in some form.